styles/grid-system.css

```diff
@@ -9,6 +9,8 @@
   .article-grid > * {
     /* The default is spanning the body for any child component */
     @apply col-body;
+    /* Grids do not have margin-collapse, so each direct child needs to be addressed */
+    margin-top: 0 !important;
   }
```

Comment:
@tavin, this is basically the fix. The rest is making sure that callouts/equations are balanced and have the same margins as paragraphs.

Reply:
understood :+1: